.travis.yml

```diff
@@ -8,6 +8,7 @@ env:
     - TRAVIS_NODE_VERSION="8"
   matrix:
     - CI_TEST: test
+    - CI_TEST: scalafmt
```

Comment:
--test is passed here https://github.com/scalameta/metadoc/blob/34070c78b8a06c885e6bef844309c7b74904127e/bin/runci.sh#L7